### PR TITLE
update npm version on CDN example to latest

### PR DESCRIPTION
**Pull request by beenotung**
## Pull request checklist

Please check if your PR fulfills the following requirements:
- [ ] Tests for the changes have been added (for bug fixes / features)
- [ ] Docs have been reviewed and added / updated if needed (for bug fixes / features)
- [ ] Build (`npm run build`) was run locally and any changes were pushed
- [ ] Lint (`npm run lint`) has passed locally and any fixes were made for failures

## Pull request type

 

 

Please check the type of change your PR introduces:
- [ ] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [x] Documentation content changes
- [ ] Other (please describe): 

## What is the current behavior?

Issue Number: N/A

## What is the new behavior?

-
-
-

## Does this introduce a breaking change?

- [ ] Yes
- [ ] No
- [x] Maybe

### CODE REVIEW

File: core/README.md [hunk 1]
```diff
@@ -23,9 +23,9 @@ The Ionic Core package contains the Web Components that make up the reusable UI
 Easiest way to start using Ionic Core is by adding a script tag to the CDN:
 
 ```html
-<script type="module" src="https://cdn.jsdelivr.net/npm/@ionic/core@4.6.2/dist/ionic/ionic.esm.js"></script>
-<script nomodule src="https://cdn.jsdelivr.net/npm/@ionic/core@4.6.2/dist/ionic/ionic.js"></script>
-<link href="https://cdn.jsdelivr.net/npm/@ionic/core@4.6.2/css/ionic.bundle.css" rel="stylesheet">
+<script type="module" src="https://cdn.jsdelivr.net/npm/@ionic/core@5.0.1/dist/ionic/ionic.esm.js"></script>
```
Comment (brandyscarney) [inline core/README.md]:
```suggestion
<script type="module" src="https://cdn.jsdelivr.net/npm/@ionic/core/dist/ionic/ionic.esm.js"></script>
```

File: core/README.md [hunk 2]
```diff
@@ -23,9 +23,9 @@ The Ionic Core package contains the Web Components that make up the reusable UI
...
+<script nomodule src="https://cdn.jsdelivr.net/npm/@ionic/core@5.0.1/dist/ionic/ionic.js"></script>
```
Comment (brandyscarney) [inline core/README.md]:
```suggestion
<script nomodule src="https://cdn.jsdelivr.net/npm/@ionic/core/dist/ionic/ionic.js"></script>
```

File: core/README.md [hunk 3]
```diff
@@ -23,9 +23,9 @@ The Ionic Core package contains the Web Components that make up the reusable UI
...
+<link href="https://cdn.jsdelivr.net/npm/@ionic/core@5.0.1/css/ionic.bundle.css" rel="stylesheet">
```
Comment (brandyscarney) [inline core/README.md]:
```suggestion
<link href="https://cdn.jsdelivr.net/npm/@ionic/core/css/ionic.bundle.css" rel="stylesheet">
```

**Comment by beenotung** [comment]
Using the latest version is fine in most cases but will it pull the -alpha or -next version when it's not ready?

**Comment by brandyscarney** [comment]
@beenotung No it would always pull in `@ionic/core@latest`, any release candidate version is published under the `@ionic/core@next` tag. 🙂

**Comment by brandyscarney** [comment]
Thanks! 🙂

**Comment by ali-master** [comment]
Into the production build, Why ionic/core uses its dependencies via jsdelivr? Because we faced a problem that affected our Banking application by this feature that have written by @ionic/react.

What's your suggestion to fix this problem? to load all dependencies locally!

**Pull request merged**